substrate/state-machine/src/ext.rs

```diff
@@ -123,7 +136,7 @@ impl<'a, B: 'a> Externalities for Ext<'a, B>
 		42
 	}
 
-	fn storage_root(&mut self) -> [u8; 32] {
+	fn storage_root(&mut self) -> H::Out where H::Out: Ord + Encodable {
```

Comment:
I know this line isn't super long, however I think it reduces readability complexity if we had the `where` in the second line